LaLauncher/src/main/java/com/github/postapczuk/lalauncher/MainActivity.java

```diff
@@ -108,6 +111,14 @@ public void fetchAppList() {
     @Override
     public void onClickHandler() {
         listView.setOnItemClickListener((parent, view, position, id) -> {
+            TextView selectedItem = view.findViewById(view.getId());
```

Comment:
Ok, now it's trippled - as I mentioned create method in AppsActivity accessible for all classes that extend it